artemis-core-client/src/main/java/org/apache/activemq/artemis/core/protocol/core/impl/wireformat/ActiveMQExceptionMessage.java

```diff
@@ -68,10 +68,7 @@ protected String getPacketString() {
 
    @Override
    public int hashCode() {
-      final int prime = 31;
-      int result = super.hashCode();
-      result = prime * result + ((exception == null) ? 0 : exception.hashCode());
-      return result;
+      return Objects.hash(super.hashCode(), exception);
```

Comment:
This is an odd one, computing the hash of the `super.hashCode()` vs just addition of that value with the exception hash ?  `super.hashCode() + Objects.hashCode(exception)`